ports/lunasvg/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "lunasvg",
-  "version": "2.3.1",
+  "version": "2.3.1+20230331",
```

A couple of comments regarding the versioning of this port.

* Using the build tag (`+<build>`) is not a good idea. 
According to SemVer, build tags have no semantic meaning, e.g.: `2.3.1` is the same version as `2.3.1+20230331`; but in practice, vcpkg must treat them differently to avoid non-determinism when resolving versions. For this reason, I believe it is better to use the same base `version` and just increases the `port-version` field to communicate that there has been an "improvement". 
* Regarding specifically to `lunasvg`, the latest release tag is `2.3.5` (released 7 months ago), and the [version in their CMakeLists.txt reflects that](https://github.com/sammycage/lunasvg/blob/f924651b85cac47dbe15f51a4aa320461fc1d07b/CMakeLists.txt#L3). So, keeping `2.3.1` as the base version while using the latest commit is wrong.

```suggestion
  "version": "2.3.5",
```

---

@vicroms, Thanks for your suggesstion, version 2.3.5 does not include usage-related changes. Currently we have two solutions. The first is to use "version-date" to update to the latest commit, and the other is to update to the latest release version 2.3.5 and use patches to add usage, which one should we use?

---

I'd like to avoid changing the version scheme unless we know that upstream is not planning to release any new version tags. For that reason, I'd like to go with the approach of using patches to backport the CMake integration into the 2.3.5 release.

@sammycage are any more `lunasvg` releases planned in the near future?